gips/data/core.py

```diff
@@ -661,17 +706,28 @@ def ParseAndAddFiles(self, filenames=None):
                         raise Exception('Mismatched dates: %s' % ' '.join(filenames))
                 sensor = parts[1 + offset]
                 product = parts[2 + offset]
-                self.AddFile(sensor, product, f)
+                self.AddFile(sensor, product, f, add_to_db=False)
             except Exception:
                 # This was just a bad file
                 VerboseOut('Unrecognizable file: %s' % f, 3)
                 continue
 
-    def AddFile(self, sensor, product, filename):
-        """ Add products (dictionary  (sensor, product): filename) to instance """
+    def AddFile(self, sensor, product, filename, add_to_db=True):
```

Comment:
yes:  default taken by `process()` in various classes, and `add_to_db=False` is passed in during `DataInventory` instantiation.